documentation/deliverables/individual_deliverables/open-source-group-kickoff.md

```diff
@@ -0,0 +1,148 @@
+# Open Source Group Kickoff
```

Comment:
```suggestion
# Open Source Onboarding Process
```
Since we're mainly focusing on implementing the comms tools for open source group and setting up the process to onboard new open source participants, I wonder if this is a more accurate title?